src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -809,7 +809,7 @@ private native boolean openPairingWindowWithPIN(
       int duration,
       long iteration,
       int discriminator,
-      long setupPinCode);
+      Long setupPinCode);
```

Comment:
for consistency, consider to use long?

Reply:
setupPinCode should be noted with @nullable, then you can use Long?

Reply:
I added `@Nullable`. Thanks.